### ISSUE TITLE
error running module suma.py AttributeError: 'tuple' object has no attribute 'rstrip'

### ISSUE THREAD

**Issue by kvangogh** [IBM/ansible-power-aix#42]
After applying suggested fix (a382be1) 

error when running the same playbook calling suma.py module to update tl levels on aix endpoint

Error details:
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'tuple' object has no attribute 'rstrip'
fatal: [ansible-vm1]: FAILED! => changed=false
  module_stderr: |-
    Shared connection to ansible-vm1 closed.
  module_stdout: |-
    Traceback (most recent call last):
      File "/.ansible/tmp/ansible-tmp-1600983454.4765153-6132-213536560555233/AnsiballZ_suma.py", line 102, in <module>
        _ansiballz_main()
      File "/.ansible/tmp/ansible-tmp-1600983454.4765153-6132-213536560555233/AnsiballZ_suma.py", line 94, in _ansiballz_main
        invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
      File "/.ansible/tmp/ansible-tmp-1600983454.4765153-6132-213536560555233/AnsiballZ_suma.py", line 40, in invoke_module
        runpy.run_module(mod_name='ansible_collections.ibm.power_aix.plugins.modules.suma', init_globals=None, run_name='__main__', alter_sys=True)
      File "/opt/freeware/lib/python2.7/runpy.py", line 176, in run_module
        fname, loader, pkg_name)
      File "/opt/freeware/lib/python2.7/runpy.py", line 82, in _run_module_code
        mod_name, mod_fname, mod_loader, pkg_name)
      File "/opt/freeware/lib/python2.7/runpy.py", line 72, in _run_code
        exec code in run_globals
      File "/tmp/ansible_suma_payload_iyhBnT/ansible_suma_payload.zip/ansible_collections/ibm/power_aix/plugins/modules/suma.py", line 869, in <module>
      File "/tmp/ansible_suma_payload_iyhBnT/ansible_suma_payload.zip/ansible_collections/ibm/power_aix/plugins/modules/suma.py", line 859, in main
      File "/tmp/ansible_suma_payload_iyhBnT/ansible_suma_payload.zip/ansible_collections/ibm/power_aix/plugins/modules/suma.py", line 676, in suma_download
    AttributeError: 'tuple' object has no attribute 'rstrip'
  msg: |-
    MODULE FAILURE
    See stdout/stderr for the exact error
  rc: 1

PLAY RECAP *************************************************************************************************************
ansible-vm1                : ok=3    changed=1    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

**Comment by robinvy** [comment]
I don't see any reference to `tuple` object in suma.py file. Please check you get an up to date version of the file.

In addition, note that the fix mentioned (a382be1) is not related to issue #40, it's related to TravisCI build.
You should apply commit `suma: fix issue #40 unpack error calling suma_command` with hash (afc340d) that is part of the last PR #39.

**Comment by aklyachkin** [comment]
Just got the same error with the version 1.1.0 of the collection.

The problem is, that the function suma_command() returns 2 values:

```
    return rc, stdout
```

but the calls to the function in the module expect just stdout:

```
  +669      stdout = suma_command('Preview')
  +708          stdout = suma_command('Download')
```

If you change it to:

```
  rc, stdout = suma_command...
```

everything works correct.

Test playbook:

```
- hosts: all
  collections:
    - ibm.power_aix
  vars:
    ansible_python_interpreter: /opt/freeware/bin/python
  tasks:
    - name: download and install updates for the current oslevel
      suma:
        action: download
        oslevel: latest
        download_dir: /usr/sys/inst.images
        extend_fs: yes
```

**Comment by robinvy** [comment]
Oh yes, my bad. I didn't mean to add the rc in the suma.py file.
`    return rc, stdout`
Just removed the rc as: `return stdout` .
The `rc != 0` case if handled in suma_comand calling `module.fail_json`. And suma_comand will always return 0 so we can avoid returning rc.
It will be fixed in the next PR #45.

**Comment by pbfinley1911** [comment]
Fixed in next collection refresh. (Oct 24th)
